std/zip.d

```diff
@@ -242,7 +242,7 @@ final class ArchiveMember
     @property @safe void expandedData(ubyte[] ed)
     {
         _expandedData = ed;
-        _expandedSize  = to!uint(_expandedData.length);
+        _expandedSize  = to!ulong(_expandedData.length);
```

Comment:
Future-proofing for 128 bit size_t?

Reply:
I can only assume this is a joke

Reply:
It was probably part of a global search/replace.

Reply:
Whatever I was thinking when I wrote the comment, it can be dismissed. I am leaving this conversation marked "resolved" rather than deleting the original comment outright because I think it would be poor form to delete a comment that had been replied to.

Reply:
I think it's valid to question this, because technically the `to` call does nothing except return its input. My point was simply that it's likely @berni44 just did a global search and replace, and this one got caught. I think you can remove the `to` call.